pilot/pkg/kube/inject/testdata/webhook/TestWebhookInject_no_volumes.patch

```diff
@@ -17,10 +14,7 @@
     "value": {
       "name": "istio-proxy",
       "image": "example.com/proxy:latest",
-      "resources": {},
-      "terminationMessagePath": "/dev/termination-log",
-      "terminationMessagePolicy": "File",
-      "imagePullPolicy": "Always"
+      "resources": {}
```

Comment:
why not remove this as well?

Reply:
You  can see the inject template https://github.com/istio/istio/blob/master/pilot/pkg/kube/inject/mesh.go#L78, 
we set `resources: {}` for container

Reply:
I mean the default is already {}

Reply:
@hzxuzhonghu I just see the test and it is not used the template I write in the link, it is use the 

```
initContainers:
- name: istio-init
  image: example.com/init:latest
containers:
- name: istio-proxy
  image: example.com/proxy:latest
imagePullSecrets:
- name: istio-image-pull-secrets
volumes:
- emptyDir:
    medium: Memory
  name: istio-envoy
- name: istio-certs
  secret:
    [[ if eq .Spec.ServiceAccountName "" -]]
    secretName: istio.default
    [[ else -]]
    secretName: [[ printf "istio.%s" .Spec.ServiceAccountName ]]
    [[ end -]]
```
why do not removed the `"resources": {}` is that the resources in container is a struct field not a point so when json.Marshal, the result is  `"resources": {}` not empty.

Reply:
ic